X7.lua

```diff
@@ -62,7 +62,21 @@ SetupPages = {
          { t = "Dev",  x = 60, y = 12, sp = 34, i=1, ro=true, table = {[3]="SA",[4]="Tramp",[255]="None"} },
          { t = "Freq", x = 60, y = 22, sp = 34, i="f", ro=true },
       },
+   },
+   -- TODO fix the alignment
+   {
+      title = "Camera OSD control",
+      text = {
+         { t = "MENU", x = 86, y = 16 },
```

Comment:
i have adjusted this to :
text = {
         { t = "MENU", x = 54, y = 12 },
         { t = "EXIT", x = 55, y = 36 },
         { t = "+", x = 32, y = 24 },
         { t = "-", x = 92, y = 24 },
         { t = "ENT", x = 56, y = 24 },
         { t = "(Hold MENU to exit page)", x = 3, y = 46 }
      },